src/index.mako.html

```diff
@@ -420,17 +420,9 @@
 
         <!-- Catalog -->
         <div class="panel" ng-hide="globals.isDrawActive">
-  % if device == 'desktop':
-          <div class="theme-toggle">
-            <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-              <span translate>choose_theme</span>
-              <i class="fa fa-chevron-sign-right"></i>
-            </a>
-            <div class="dropdown-menu">
-              <div ga-topic ga-topic-ui="list"></div>
-            </div>
-          </div>
-  % endif
+          <button type="button" class="btn btn-default pull-right topic-btn" data-toggle="modal" data-target=".ga-topic-modal">
```

Comment:
can you use an identifier for the data-target like this `data-target="#topicSelector .modal"` ?   A class could potentially be not unique.